client/src/components/History/Content/ContentItem.vue

```diff
@@ -150,7 +150,7 @@ export default {
         },
         state() {
             if (this.item.job_state_summary) {
-                for (const key of ["error", "failed", "paused", "upload", "running"]) {
+                for (const key of STATES_REDUCE) {
                     if (this.item.job_state_summary[key] > 0) {
                         return key;
```

Comment:
But can't you just replace the whole function with `this.item.job_state_summary.state` ?

Reply:
ok, the job_summary_state seems like the right abstraction, but it also looks very suspicious. Lot of the methods don't seem to work and are not actually used. It would be fantastic if this could be unified.